00-overall/02-header/header.css

```diff
@@ -188,72 +174,134 @@
   font-weight: 400;
 }
 
-.header-login-container {
+.header-mobile-login-container {
   display: none;
 }
 
-.header-search-container{
+.header-mobile-search-container{
   display: none;
 }
 
-/*
--- CLASS NAMES AND LAYOUT HAVE BEEN CHANGED SINCE THIS --
+.header-desktop-container{
+  display: none;
+}
 
-CSS FROM homepage-desktop.css 
+/*****Header Desktop*******/
+@media (min-width:600px){
+.header-mobile-container{
+  display: none;
+}
+.header-desktop-container,
+.header-desktop-logo-container,
+.header-desktop-nav-container,
+.header-desktop-nav-list,
+.header-desktop-login-container,
+.header-desktop-search-container{
+  border: var(--temporary-border);
+}
 
-.header-container {
+.header-desktop-container{
   display: grid;
-  grid-template: repeat(2,auto) / repeat(8,1fr);
-  background-color: var(--light-perimary);
-  width: 100%;
-  font-size: 2rem;
+  grid-template-columns: 2fr 5fr 1fr 2fr;
+  grid-template-rows: repeat(3,auto);
+  grid-template-areas: 
+  "header-desktop-logo-container . .  header-desktop-login-container "
+  "header-desktop-logo-container . . header-desktop-search-container"
+  ". header-desktop-nav-container . .";
+  background-color: var(--dark-blue);
 }
 
-.logo-container { 
-  grid-column: 1/2;
-  grid-row: 1/3;
-  justify-self: start;
+/****LOGO CONTAINER****/
+
+.header-desktop-logo-container {
+ grid-area: header-desktop-logo-container;
   align-self: center;
+  text-align: center;
 }
 
-.logo-text{
-  background-color: var(--white);
+.header-desktop-logo-image-container {
+  display: flex;
+  justify-content: center;
+  gap: 12px; 
 }
 
-.nav-header-container {
-  grid-column: 2/7;
-  grid-row: 1/3;
-  list-style: none;
-  width:100%;
-  justify-self: start;
-  align-self: center;
+.header-desktop-logo-image {
+  width: 16px;
+  height: 22px;
 }
 
-.header-nav-list {
+.header-desktop-logo {
+  color: var(--red);
+  font-size: 20px;
+  font-weight: 800;
+}
+
+.header-desktop-logo-highlight {
+  color: var(--white);
+  font-weight: 600;
+}
+
+/******NAV CONTAINER*****/
+
+.header-desktop-nav-container{
+  grid-area: header-desktop-nav-container;
+  justify-content: center;
+}
+
+.header-desktop-nav-list{
+ display: flex;
+ flex-direction: row;
+ justify-content: space-evenly;
```

Comment:
@karleenmsrichards this is why the links don't start at the first i can change this to space between and home start near the logo but may it cause more space between the links. the other way is that we can make the flex start and add padding to every link. i will change it to space between for now to see how it looks.